Sources/StringExtensions.swift

```diff
@@ -450,11 +450,15 @@ extension String {
         return false
     }
     
+    #if os(iOS)
```

Comment:
Lets add a comment on top of this as to why this change is being made ("UIPasteboard is __TVOS_PROHIBITED and hence only supported in iOS)"